src/protocols/kubernetes/settings.c

```diff
@@ -418,6 +425,19 @@ guac_kubernetes_settings* guac_kubernetes_parse_args(guac_user* user,
         guac_user_parse_args_int(user, GUAC_KUBERNETES_CLIENT_ARGS, argv,
                 IDX_BACKSPACE, GUAC_TERMINAL_DEFAULT_BACKSPACE);
 
+    /* Set the maximum number of bytes to allow within the clipboard. */
+    settings->clipboard_buffer_size =
+        guac_user_parse_args_int(user, GUAC_KUBERNETES_CLIENT_ARGS, argv,
+                IDX_CLIPBOARD_BUFFER_SIZE, 0);
+
+    /* Use default clipboard buffer size if given one is invalid. */            
+    if (settings->clipboard_buffer_size <= 0) {
```

Comment:
Shouldn't we set to `GUAC_COMMON_CLIPBOARD_MAX_LENGTH` if `settings->clipboard_buffer_size > GUAC_COMMON_CLIPBOARD_MAX_LENGTH`

Reply:
Fixed. Added max/min clipboard size limits.